frontend/lib/norent/letter-builder/know-your-rights.tsx

```diff
@@ -83,8 +88,13 @@ export const StateWithProtectionsContent: ProtectionsContentComponent = (
     <p>{props.lawForBuilder.textOfLegislation}</p>
     <p>
       We’ve partnered with{" "}
-      <PartnerLink {...(props.partner || DefaultStatePartnerForBuilder)} /> to
-      provide additional support once you’ve sent your letter.
+      {props.partner && (
+        <>
+          <PartnerLink {...props.partner} /> and{" "}
+        </>
+      )}
+      <PartnerLink {...DefaultStatePartnerForBuilder} /> to provide additional
+      support once you’ve sent your letter.
```

Comment:
Ugh, copying this logic definitely wasn't DRY but since it's such a small snippet and it relied on the main props object, I just left it as is to avoid more complexity here...

Reply:
Also, full disclosure, I'm tired AF

Reply:
hehe no worries!  I filed #1335 to address it later.